design/www/app/login/login.controller.js

```diff
@@ -33,7 +33,7 @@
                     console.log(data);
                     if(data.status == 0){
                         $state.go('verification');
-                        timeStorage.set('userEmail',this.data.email);
+                        timeStorage.set('userEmail',this.data.email, 1);
```

Comment:
Sir in this point if user try to login and if response is status == 0 means user is not verified his account so redirect to verification page and i saved this email id for verification page. if email is saved user didn't need to enter his email again on verification page.